src/Motor.Extensions.Hosting.RabbitMQ/RabbitMQMessageHostBuilderExtensions.cs

```diff
@@ -2,9 +2,12 @@
 using CloudNative.CloudEvents.SystemTextJson;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using MSOptions = Microsoft.Extensions.Options.Options;
```

Comment:
mini nit: I usually expect these kind of `using`s at the bottom of the `using` statements but as long as `dotnet format` is fine with it my ocd has to be quiet :smile: